packages/react-i18n/src/i18n.ts

```diff
@@ -43,12 +44,24 @@ const RTL_LANGUAGES = [
 /* eslint-enable */
 
 export default class I18n {
-  locale: string;
-  defaultCurrency?: string;
-  defaultTimezone?: string;
+  readonly locale: string;
+  readonly pseudolocalize: boolean | string;
+  readonly defaultCurrency?: string;
+  readonly defaultTimezone?: string;
 
-  get language(): string {
-    return this.locale.split('-')[0];
+  get language() {
+    return languageFromLocale(this.locale);
+  }
+
+  get region() {
+    return regionFromLocale(this.locale);
+  }
+
+  /**
+   * @deprecated Use I18n#region instead.
+   */
+  get countryCode() {
```

Comment:
this feels a bit weird to do in this PR

Reply:
I definitely see that, but it was a choice between keeping the incorrect language for pseudo localization as well (where we use the region), or updating them both.